faker/compat.py

```diff
@@ -1,6 +1,6 @@
 # Python 2/3 compat
 import sys
-if sys.version < '3':
+if sys.version_info[0] < 3:
```

Comment:
Using `sys.version_info.major` doesn't work for Python 2.6 and below though. So it's more readable but it would restrict the Python versions to Python 2.7+.